drivers/at24mac/Makefile.dep

```diff
@@ -1 +1,7 @@
-USEMODULE += at24cxxx
+# Deprecated functionality, the eeprom should be independent of the id.
+# Should be removed in the 2022.07 release.
+USEMODULE += at24mac_eeprom
```

Comment:
```suggestion
```

Probably we should not use the pseudomodule by default because the `at24c*` pseudomodules exist for the reason mentioned above.

Reply:
I want to remove it, but we want to keep the same functionality as before.  If someone would `USEMODULE += at24mac` it would select at24mac variant.

After the depreciation period we will.

Reply:
Ah yes true, it´s removed afterwards.